code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -203,6 +228,7 @@
 	toggleable = TRUE
 	use_power_cost = 0.4
 	passive_power_cost = 0.4
+	backstab_damage = 6
```

Comment:
why only 6?

Reply:
A bluespace dagger isn't supposed to damage you, it's supposed to embed into the target and then teleport them away.